run_tests.sh

```diff
@@ -1,3 +0,0 @@
-#! /bin/sh
-
-nosetests --verbose --with-doctest --with-coverage --cover-package pydocx $@ && find pydocx -name '*.py' | xargs flake8
```

Comment:
Why did you remove this file? I really like having a one liner that runs the tests faster than tox does.